code/modules/lighting/lighting_setup.dm

```diff
@@ -1,8 +1,11 @@
 /proc/create_all_lighting_objects()
+/proc/create_lighting_objects_zlevel(zlevel)
+
+/create_all_lighting_objects()
```

Comment:
The proc isn't called or breaks, as lighting_objects are never created, so it does not work.